### DISCUSSION TITLE
Enum stuff

### DISCUSSION THREAD

**Discussion by STPv22** [eaglerforge/EaglerForgeInjector#45] [Q&A] [answered]
![Screenshot 2024-12-10 5 47 45 PM](https://github.com/user-attachments/assets/1f71ed34-8050-4325-93d8-1d01cf73ec96)
trying to make an item like a bow and am not sure how to go about adding and accessing enum stuff
![image](https://github.com/user-attachments/assets/fc9a1dce-da56-4da4-96c8-fc05827e3368)

![image](https://github.com/user-attachments/assets/1dbdd252-f480-4a45-9088-a73d96021a60)

**Comment by STPv22** [comment]
lmao I'm retarted nvm

**Comment by STPv22** [comment]
nevermind I need help

**Comment by radmanplays** [comment]
@ZXMushroom63

**Comment by ZXMushroom63** [comment, marked as answer]
Get the enum using ModAPI.reflect.getClassByName("EnumAction"). staticVariables.DRINK

**Comment by STPv22** [comment]
okeee

**Comment by STPv22** [comment]
thx
